OJP/OJP_JourneySupport.xsd

```diff
@@ -231,7 +231,7 @@
 					<xs:documentation>Indicates whether this service needs some kind of reservation or booking to run.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded"/>
+			<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0"/>
```

Comment:
annotation documentation is missing on all these `SituationFullRefs`.

Reply:
changed